### PR TITLE
[SPARK-10249][ML][Doc]Add Python Code Example to StopWordsRemover User Guide

### PR DESCRIPTION
**Pull request by hhbyyh**
jira: https://issues.apache.org/jira/browse/SPARK-10249

update user guide since python support added.

### CODE REVIEW

File: docs/ml-features.md [hunk 1]
```diff
@@ -512,6 +512,25 @@ DataFrame dataset = jsql.createDataFrame(rdd, schema);
...
+{% highlight python %}
+from pyspark.ml.feature import StopWordsRemover
+
+sentenceData = sqlContext.createDataFrame([
+  (0, ["I", "saw", "the", "red", "baloon"]),
+  (1, ["Mary", "had", "a", "little", "lamb"])
+], ["label", "raw"])
+
+remover = StopWordsRemover(inputCol="raw", outputCol="filtered")
+filtered = remover.transform(sentenceData).show()
```
Comment (mengxr) [inline docs/ml-features.md]: * `show()` doesn't return anything
* minor: it would be useful to call `show()` with `truncate=False` for user guide.

**Comment by mengxr** [comment]
LGTM except one minor inline comment

**Comment by hhbyyh** [comment]
Thanks for the review @mengxr

**Comment by mengxr** [comment]
Merged into master. Thanks!

**Pull request closed without merging**